lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/ReadOnlySystemAccessControl.java

```diff
@@ -153,6 +153,12 @@ public Set<String> filterColumns(SystemSecurityContext context, CatalogSchemaTab
         return columns;
     }
 
+    @Override
+    public Map<SchemaTableName, Set<String>> filterColumns(SystemSecurityContext context, String catalogName, Map<SchemaTableName, Set<String>> tableColumns)
```

Comment:
Should we deprecate the other `filterColumns` method?

Reply:
done